### PR TITLE
Fix typos and unused imports

### PR DESCRIPTION
**Pull request by quentinlesceller**
Small PR to remove some unused imports and fix trivial typos.

### CODE REVIEW

File: chain/src/txhashset.rs [hunk 1]
```diff
@@ -476,8 +476,7 @@ impl<'a> Extension<'a> {
 		let kernel_pos = self.kernel_pmmr.unpruned_size();
 
 		// Build bitmap of output pos spent (as inputs) by this tx for rewind.
-		let rewind_rm_pos = tx
-			.inputs()
+		let rewind_rm_pos = tx.inputs()
```
Comment (antiochp) [inline chain/src/txhashset.rs]: I see PRs flip-flopping on this code style _all the time_ (first chained fn call on same line or next line).
Is this a difference between local rustfmt configs?

I'm on - 
```
rustfmt --version
rustfmt 0.8.2-stable (08da30d7 2018-06-06)
```

And I _think_ my rustfmt puts these on the next line which contradicts this PR.

File: core/tests/consensus.rs [hunk 2]
```diff
@@ -137,7 +139,8 @@ fn get_diff_stats(chain_sim: &Vec<Result<(u64, Difficulty), TargetError>>) -> Di
 
 	// Obtain the median window for the latest time period
 	// i.e. the last  MEDIAN_TIME_WINDOW elements
-	let mut window_latest: Vec<u64> = last_blocks.clone()
```
Comment (antiochp) [inline core/tests/consensus.rs]: But then here it appears to be the opposite... (was on same line, now on next line).

Comment (sesam) [reply core/tests/consensus.rs]: :+1: for a whole-project rustfmt commit soon-ish. Then if rustfmt issues continue to pop up, maybe check tab size configs and maybe use .editorconfig in project root

File: chain/src/txhashset.rs (hunk 1 above)
Comment (quentinlesceller) [reply chain/src/txhashset.rs]: Agree, I think this is something related to https://github.com/rust-lang-nursery/rls-vscode/issues/210. Investigating.

Comment (antiochp) [reply chain/src/txhashset.rs]: :+1:

**Pull request merged**